pages/ner.py

```diff
@@ -18,29 +20,29 @@
 model_id = 'ner-english'
 model_version_id = 'a813ff5b362c41f790c506b871e7dea4'
```

Comment:
Can we directly use something like this `model_prediction = Model("https://clarifai.com/anthropic/completion/models/claude-v2").predict_by_bytes(b"Write a tweet on future of AI", "text")`